### ISSUE TITLE
Native Base Tabs RTL support issue in android

### ISSUE THREAD

**Issue by ramkgunturu** [GeekyAnts/NativeBase#3504]
When I placed nested tabs(Parent and Child Tabs)  in one screen,when i swipe the screen child tabs will swipe until it completes the loops,And then it will move to the parent Tab .

It works very well in IOS but on android.

In Android,  first tab data is not loading in RTL, it is showing empty screen.

Could anyone please help me on the above two issue will be a great help!.
<img width="332" alt="Untitled" src="https://user-images.githubusercontent.com/58797826/109498290-44fe7f80-7ab9-11eb-88e9-f8028e6455da.png">

**Comment by ansarikhurshid786** [comment]
Use page property to fixed issue in android: 
```
this.state = {
bottomDrawerTabPage: 0
}

<Tabs 
   initialPage={0} 
   page={this.state.bottomDrawerTabPage}
   onChangeTab={({ from, i }) => this.setState({ bottomDrawerTabPage: i })}
>
...
</Tabs>
```

This works for me

**Comment by ansarikhurshid786** [comment]
Following stackoverflow fixed for me: 
https://stackoverflow.com/questions/58736619/nativebase-tabs-issue-for-force-rtl-fix/58736621#58736621

**Comment by ramkgunturu** [comment]
If we place reverse then swipe functionality is not changing.

Client needs swipe functionality to change tabs.
